var/spack/repos/spack_repo/builtin/packages/mgard/package.py

```diff
@@ -10,68 +10,92 @@
 
 class Mgard(CMakePackage, CudaPackage):
     """MGARD error bounded lossy compressor
-    forked from https://github.com/CODARcode/MGARD with patches to support Spack"""
 
-    # This is a research compressor with a fast evolving API.  The fork is updated
-    # when releases are made with minimal changes to support spack
+    For versions up to 2023-12-09, uses a fork from https://github.com/robertu94/MGARD
+    with patches to support Spack. For versions after 2023-12-09, uses the upstream
+    repository at https://github.com/CODARcode/MGARD directly from the master branch.
+    """
+
+    # This is a research compressor with a fast evolving API. Historical versions
+    # use a fork with minimal changes to support spack, while new versions use upstream master.
```

Comment:
This comment should be removed. MGARD is no longer a fast evolving API. As I write this, the last change was was 3 months ago, and that was to support this spack package.

Reply:
I agree this can go